sandbox/gamedev/unity/samples/azure-facebook-auth-unity.md

```diff
@@ -0,0 +1,646 @@
+---
+title: "Facebook authentication with Unity and Azure"
+author: dantogno
+ms.author: v-davian
+ms.date: 01/24/2018
+ms.topic: article
+ms.assetid:
+---
+# Facebook authentication with Unity and Azure
+
+![Facebook authentication with Unity and Azure heading image](media/fbauth_fb-auth-heading.png)
+
+## Overview
+
+This documentation demonstrates how to incorporate Facebook authentication into a Unity project that stores high score and game telemetry data in Azure Easy Tables.
+
+The approach uses the [Facebook Unity SDK](https://developers.facebook.com/docs/unity/) to allow users to log in to Facebook. It then uses [UnityWebRequest](https://docs.unity3d.com/Manual/UnityWebRequest.html) to send HTTP requests to an [Azure function app](https://azure.microsoft.com/en-us/services/functions/) that handles authentication and data insertion and retrieval.
+
+### Compatibility
+
+This approach should work for any Unity platform supported by the Facebook SDK.
+* iOS
+* Android
+* WebGL
+
+> [!NOTE]
+> PC builds can be supported by using the [Facebook Gameroom](https://unity3d.com/partners/facebook/gameroom) build feature of Unity.
+
+### Prerequisites
+
+* Unity 2017 ([free personal version](https://store.unity.com/products/unity-personal) is fine)
+* Azure subscription (the [free trial](https://azure.microsoft.com/en-us/free/) or included credits from [Visual Studio Dev Essentials](https://www.visualstudio.com/dev-essentials/) are fine)
+* [Visual Studio 2017](https://www.visualstudio.com/) (free Community edition is fine) with Game Development with Unity workload
+* Facebook account
+
+## Create an Azure Mobile App and configure Easy Tables
+
+An [Azure Mobile App](https://azure.microsoft.com/en-us/services/app-service/mobile) is a type of Azure App Service. This example uses the Easy Tables feature of Azure Mobile Apps to simply store data.
```

Comment:
Point to Docs link https://azure.microsoft.com/en-us/documentation/learning-paths/appservice-mobileapps/